ctera/src/main/java/ch/cyberduck/core/ctera/CteraAttributesFinderFeature.java

```diff
@@ -0,0 +1,54 @@
+package ch.cyberduck.core.ctera;
+
+import ch.cyberduck.core.Acl;
+import ch.cyberduck.core.Path;
+import ch.cyberduck.core.PathAttributes;
+import ch.cyberduck.core.Permission;
+import ch.cyberduck.core.dav.DAVAttributesFinderFeature;
+import ch.cyberduck.core.dav.DAVPathEncoder;
+import ch.cyberduck.core.dav.DAVSession;
+
+import javax.xml.namespace.QName;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import com.github.sardine.DavResource;
+import com.github.sardine.util.SardineUtil;
+
+import static ch.cyberduck.core.ctera.CteraAclPermissionFeature.*;
+
+public class CteraAttributesFinderFeature extends DAVAttributesFinderFeature {
```

Comment:
Change test `CteraAttributesFinderFeatureTest` to use this implementation.

Reply:
034506860cd4d01e3d6f8b919b94e2e620bf3f29